docs/contents/user-docs/irsa.md

```diff
@@ -31,8 +31,150 @@ to assume the IAM Role with reduced permissions.
     amazon-eks-pod-identity-webhook repository for setting up IRSA on your own
     Kubernetes installation.
 
+## IRSA setup on EKS cluster and install ACK controller using helm
+Following steps provide example to setup IRSA on EKS cluster to install ACK ElastiCache controller using helm charts.
+By modifying the variables values as needed, these steps can be applied for other ACK controllers.
+
+The steps include:
+
+### 1. Create OIDC identity provider for cluster**
```

Comment:
Remove the `**` at the end of the section header (it triggers bold font in Markdown until a following `**`)